drivers/ukbus/pci/include/uk/bus/pci.h

```diff
@@ -298,8 +470,39 @@ static struct pci_bus_handler ph __unused;
 #define PCI_MIN_GNT		0x3e	/* 8 bits */
 #define PCI_MAX_LAT		0x3f	/* 8 bits */
 
+/* PCI capabilities */
+#define PCI_CAP_MSIX		0x11
+
 struct pci_driver *pci_find_driver(struct pci_device_id *id);
 
+int pci_generic_config_read(__u8 bus, __u8 devfn,
+			    int where, int size, void *val);
+
+int pci_generic_config_write(__u8 bus, __u8 devfn,
+			     int where, int size, __u32 val);
+/**
+ * @brief Create a memory mapping for a memory BAR
```

Comment:
I don't see the `@breif` tag used often. Is it a remnant of an old convention?

Reply:
I think it should be removed as well

@mschlumpp can you adapt the comments to the [coding-conventions](https://unikraft.org/docs/contributing/coding-conventions#function-documentation), please?

Reply:
Ah yeah sure, it's just pretty old code that predates the coding conventions.